command/benchmark/benchmark.go

```diff
@@ -0,0 +1,121 @@
+package benchmark
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/benchmark"
+	"github.com/0xPolygon/polygon-edge/benchmark/common"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/spf13/cobra"
+)
+
+const (
+	rootJSONRPCFlag  = "rootJSONRPC"
+	childJSONRPCFlag = "childJSONRPC"
+	privateKeyFlag   = "privateKey"
+)
+
+type benchmarkParams struct {
+	rootJSONRPC  string
+	childJSONRPC string
+	privateKey   string
+}
+
+type benchmarkResult struct {
+	name   string
+	result string
+}
+
+var (
+	params *benchmarkParams = &benchmarkParams{}
+)
+
+func GetCommand() *cobra.Command {
+	benchmarkCmd := &cobra.Command{
+		Use:     "benchmark-test",
+		Short:   "Run benchmark tests",
+		Example: getCmdExample(),
+		Run:     runCommand,
+	}
+
+	setFlags(benchmarkCmd)
+
+	return benchmarkCmd
+}
+
+func runCommand(cmd *cobra.Command, _ []string) {
+	outputter := command.InitializeOutputter(cmd)
+	defer outputter.WriteOutput()
+
+	// set up the testing environment
+	testing.Init()
+
+	// set testing params
+	benchmark.RootJSONRPC = &params.rootJSONRPC
+	benchmark.ChildJSONRPC = &params.childJSONRPC
+	benchmark.PrivateKey = &params.privateKey
+
+	// set up environment, get test cases and clean up fn
+	testCases, cleanUpFn := benchmark.RootChildSendTxSetUp(&testing.B{})
+	defer cleanUpFn()
+
+	// Loop over the test cases and call the benchmark test
+	for _, testInput := range testCases {
+		sendTxResult := testing.Benchmark(func(b *testing.B) {
+			b.Helper()
+			common.TxTestCasesExecutor(b, testInput)
+		})
+		benchmarkResult := &benchmarkResult{
+			name:   testInput.Name,
+			result: fmt.Sprintf("%s %s", sendTxResult.String(), sendTxResult.MemString()),
+		}
+		outputter.SetCommandResult(benchmarkResult)
+		outputter.WriteOutput()
+	}
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.rootJSONRPC,
+		rootJSONRPCFlag,
+		"",
+		"JSONRPC address of the root node",
+	)
+	cmd.Flags().StringVar(
+		&params.childJSONRPC,
+		childJSONRPCFlag,
+		"",
+		"JSONRPC address of the child node",
+	)
+	cmd.Flags().StringVar(
+		&params.privateKey,
+		privateKeyFlag,
+		"",
+		"private key that will be used to send tx (it is expected that this address has enough funds)",
+	)
+
+	_ = cmd.MarkFlagRequired(rootJSONRPCFlag)
```

Comment:
In the `README` file you are mentioning that these flags are not required, and command has a default behavior for each case. Should these `MarkFlagRequired` be removed?